pages/services/kubernetes/2.1.0-1.12.3/release-notes/index.md

```diff
@@ -15,7 +15,7 @@ excerpt: Release notes for DC/OS Kubernetes version 2.1.0-1.12.3
 * DC/OS Kubernetes `2.1.0-1.12.3` requires DC/OS 1.12.
 * DC/OS Kubernetes `2.1.0-1.12.3` introduces breaking changes to the way the package works and is deployed.
   Therefore, it is not possible to upgrade an existing installation of DC/OS Kubernetes to `2.1.0-1.12.3`.
-* Before installing `kubernetes-cluster` package `2.1.0-1.12.3`, the `kubernetes` package must be [installed and running](/services/kubernetes/2.1.0-1.12.3/getting-started/installing-mke/).
+* Before installing `kubernetes-cluster` package `2.1.0-1.12.3`, the `kubernetes` package must be [installed and running](../install-and-customize).
```

Comment:
Is this link change correct?

Reply:
it's not correct, changes have been made in this repo that wasn't present in the kubernetes repo.